[2023-05-29T06:34:59.042+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ml_pipeline.machine_learning_task manual__2023-05-29T06:34:38.722867+00:00 [queued]>
[2023-05-29T06:34:59.048+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ml_pipeline.machine_learning_task manual__2023-05-29T06:34:38.722867+00:00 [queued]>
[2023-05-29T06:34:59.048+0000] {taskinstance.py:1331} INFO - Starting attempt 1 of 1
[2023-05-29T06:34:59.056+0000] {taskinstance.py:1350} INFO - Executing <Task(PythonOperator): machine_learning_task> on 2023-05-29 06:34:38.722867+00:00
[2023-05-29T06:34:59.062+0000] {standard_task_runner.py:57} INFO - Started process 772 to run task
[2023-05-29T06:34:59.066+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'ml_pipeline', 'machine_learning_task', 'manual__2023-05-29T06:34:38.722867+00:00', '--job-id', '23', '--raw', '--subdir', 'DAGS_FOLDER/ml_pipeline.py', '--cfg-path', '/tmp/tmpv02jnuzq']
[2023-05-29T06:34:59.069+0000] {standard_task_runner.py:85} INFO - Job 23: Subtask machine_learning_task
[2023-05-29T06:34:59.109+0000] {task_command.py:410} INFO - Running <TaskInstance: ml_pipeline.machine_learning_task manual__2023-05-29T06:34:38.722867+00:00 [running]> on host 84bce60ae612
[2023-05-29T06:34:59.164+0000] {taskinstance.py:1570} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='lequanghop844@gmail.com' AIRFLOW_CTX_DAG_OWNER='Felix Le' AIRFLOW_CTX_DAG_ID='ml_pipeline' AIRFLOW_CTX_TASK_ID='machine_learning_task' AIRFLOW_CTX_EXECUTION_DATE='2023-05-29T06:34:38.722867+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-05-29T06:34:38.722867+00:00'
[2023-05-29T06:34:59.431+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/joblib/parallel.py:776: UserWarning: Loky-backed parallel loops cannot be called in a multiprocessing, setting n_jobs=1
  **self._backend_args)

[2023-05-29T06:34:59.432+0000] {logging_mixin.py:149} INFO - Fitting 3 folds for each of 1 candidates, totalling 3 fits
[2023-05-29T06:35:04.659+0000] {logging_mixin.py:149} INFO -  1/92 [..............................] - ETA: 15s
[2023-05-29T06:35:04.710+0000] {logging_mixin.py:149} INFO - 79/92 [========================>.....] - ETA: 0s 
[2023-05-29T06:35:04.718+0000] {logging_mixin.py:149} INFO - 92/92 [==============================] - 0s 651us/step
[2023-05-29T06:35:04.735+0000] {logging_mixin.py:149} INFO - [CV] END batch_size=128, dropout=0.4, epochs=50, learning_rate=0.0001, num_units=32; total time=   5.3s
[2023-05-29T06:35:09.916+0000] {logging_mixin.py:149} INFO -  1/92 [..............................] - ETA: 13s
[2023-05-29T06:35:09.966+0000] {logging_mixin.py:149} INFO - 80/92 [=========================>....] - ETA: 0s 
[2023-05-29T06:35:09.974+0000] {logging_mixin.py:149} INFO - 92/92 [==============================] - 0s 638us/step
[2023-05-29T06:35:09.984+0000] {logging_mixin.py:149} INFO - [CV] END batch_size=128, dropout=0.4, epochs=50, learning_rate=0.0001, num_units=32; total time=   5.2s
[2023-05-29T06:35:14.774+0000] {logging_mixin.py:149} INFO -  1/92 [..............................] - ETA: 14s
[2023-05-29T06:35:14.824+0000] {logging_mixin.py:149} INFO - 78/92 [========================>.....] - ETA: 0s 
[2023-05-29T06:35:14.833+0000] {logging_mixin.py:149} INFO - 92/92 [==============================] - 0s 658us/step
[2023-05-29T06:35:14.844+0000] {logging_mixin.py:149} INFO - [CV] END batch_size=128, dropout=0.4, epochs=50, learning_rate=0.0001, num_units=32; total time=   4.9s
[2023-05-29T06:35:22.539+0000] {save.py:276} WARNING - Found untraced functions such as _update_step_xla, lstm_cell_3_layer_call_fn, lstm_cell_3_layer_call_and_return_conditional_losses while saving (showing 3 of 3). These functions will not be directly callable after loading.
[2023-05-29T06:35:22.806+0000] {builder_impl.py:798} INFO - Assets written to: dags/model/best_lstm_model.SavedModel/assets
[2023-05-29T06:35:22.951+0000] {python.py:183} INFO - Done. Returned value was: None
[2023-05-29T06:35:22.960+0000] {taskinstance.py:1373} INFO - Marking task as SUCCESS. dag_id=ml_pipeline, task_id=machine_learning_task, execution_date=20230529T063438, start_date=20230529T063459, end_date=20230529T063522
[2023-05-29T06:35:23.169+0000] {local_task_job_runner.py:232} INFO - Task exited with return code 0
[2023-05-29T06:35:23.190+0000] {taskinstance.py:2674} INFO - 1 downstream tasks scheduled from follow-on schedule check
